components/Shared/AccountCard/index.jsx

```diff
@@ -16,13 +17,25 @@ import {
   IMPORT_SINGLE_KEY
 } from '../../../constants'
 
+const CopyAddress = styled(BaseButton)`
+  /* !important is declared here to override BaseButton's opacity:0.8 on hover. The only instance of us using this declaration. */
+  opacity: 1 !important;
+  border: 0;
+  background: transparent;
+  padding: 0;
+`
+
+const StyledIconCopyAccountAddress = styled(IconCopyAccountAddress)`
+  transition: 0.24s ease-in-out;
+  ${CopyAddress}:hover & {
+    transform: scale(1.25);
+  }
+`
+
 const LabelCopy = styled(Label)`
   transition: 0.18s ease-in;
   opacity: 0;
-`
-
-const CopyAddress = styled(Box)`
-  &:hover ${LabelCopy} {
+  ${CopyAddress}:hover & {
     opacity: 1;
   }
```

Comment:
@Schwartz10 I'm not happy with this implementation, but I couldn't find a way for the `IconCopyAccountAddress` to successfully target `LabelCopy` on hover, despite following styled-component's guidelines here: https://styled-components.com/docs/advanced#referring-to-other-components

My preferred implementation would have been for the `LabelCopy` button opacity to only be triggered when a user hovers over `IconCopyAccountAddress`.

Reply:
I hear ya, i checked it out and think it looks good. Good to be conscious of using !important, in this case it seems like its the right thing to use and not sink too much time into